features/FEATURE_BOOTLOADER/mbed_lib.json

```diff
@@ -0,0 +1,38 @@
+{
+    "name": "bootloader",
+    "target_overrides": {
+        "*": {
+            "target.app_offset": "0x08010400",
+            "target.header_offset": "0x08010000",
```

Comment:
These two parameters are still wrong. Should be instead:
```
+            "target.app_offset": "0x10400",
+            "target.header_offset": "0x10000",
```

Reply:
@teetak01, I based this change from the 
[https://github.com/ARMmbed/mbed-bootloader-internal/blob/master/mbed_app.json]()
Can you clarify why we should not add flash-start-address?

Reply:
I can see that the flash-start-address from [https://github.com/ARMmbed/mbed-bootloader-internal/blob/master/mbed_app.json]() is already added in the compile. I will correct the default addresses.